server/game/cards/events/01/growingstrong.js

```diff
@@ -0,0 +1,42 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../../drawcard.js');
+
+class GrowingStrong extends DrawCard {
+
+    canPlay() {
+        if(this.game.currentPhase !== 'challenge') {
+            return false;
+        }
+        return true;
+    }
+
+    play() {
+        this.waitingPrompt = 'Waiting for opponent to use ' + this.name;
+        this.game.promptForSelect(this.controller, {
+            numCards: 3,
+            activePromptTitle: 'Select Tyrell characters to buff',
+            waitingPromptTitle: 'Waiting for opponent to use ' + this.name,
+            cardCondition: card => {
+                return card.getFaction() === this.getFaction() && card.getType() === 'character';
```

Comment:
This faction check is fine currently but will be wrong as soon as #445 is merged. @cryogen 👀